### ISSUE TITLE
Charts not displaying correctly

### ISSUE THREAD

**Issue by kurtsreynolds** [fei-protocol/checkthechain#38]
This includes Volume, Market Cap, Price charts (when using `ctc cg SYMBOL`), but also various other charts e.g. gas prices. Please see pic.
![image](https://user-images.githubusercontent.com/114658317/193004552-8a939b77-9ba8-4bea-8707-174829e41672.png)

**Comment by sslivkoff** [comment]
I believe this is because that terminal font does not contain the braille characters that ctc uses for creating charts

the underlying charting [lib](https://github.com/sslivkoff/toolstr/blob/main/toolstr/charts/char_dicts.py) can output charts using other charsets. i will make those available as alternatives in a future ctc release

can you paste some unicode quadrant chars into your terminal and see if they show up or not? for example these ones: ▚ ▟ ▌▖

**Comment by detroitcoder** [comment]
Yes this an issue with your terminal being unable to display unicode.. You should be able to resolve with a quick stack overflow search. I had a simiar issue.
